### ISSUE TITLE
Choose an architecture for the model

### ISSUE THREAD

**Issue by jtorhoff** [privML/privacy-evaluator#13]
We should agree on an architecture of the model that we'll train and use for this project before proceeding with #6 and #7.

**Comment by jtorhoff** [comment]
I've found the model in [this paper](https://www.scitepress.org/Papers/2018/67520/67520.pdf) to be sensible. It resembles the architectures of other modern CNN models used for classification (blocks consisting of Convolution, BatchNorm, and ReLU). I think we can use it for this project. The paper claims 94% on CIFAR10 and I've already set up a Jupyter notebook and trained it for a bit. It achieved a little under 90% within 50 epochs.

Here is an overview:

<img width="119" alt="Model architecture" src="https://user-images.githubusercontent.com/20026551/115999281-78f2ae80-a5eb-11eb-95ae-3bdfa072f93b.png">
